xbmc/guilib/guiinfo/AddonsGUIInfo.cpp

```diff
@@ -169,6 +169,14 @@ bool CAddonsGUIInfo::GetBool(bool& value, const CGUIListItem *gitem, int context
       value = CServiceBroker::GetAddonMgr().GetAddon(info.GetData3(), addon) && addon;
       return true;
     }
+    case SYSTEM_ADDON_IS_ENABLED:
+    {
+      value = false;
+      ADDON::AddonPtr addon;
+      if (CServiceBroker::GetAddonMgr().GetAddon(info.GetData3(), addon))
+        value = !CServiceBroker::GetAddonMgr().IsAddonDisabled(info.GetData3());
```

Comment:
This "IsAddonDisabled" is by call of GetAddon not direct required. GetAddon give normally without the fourth value only a enabled addon and returns false if not present.

Maybe a more clear call can be:
```cpp
value = (CServiceBroker::GetAddonMgr().IsAddonInstalled(info.GetData3()) &&
        !CServiceBroker::GetAddonMgr().IsAddonDisabled(info.GetData3())
```

*Just a note:* Want to replace this IsAddonDisabled in future to a IsAddonEnabled who is better for me.